etcdctl/ctlv3/command/migrate_command.go

```diff
@@ -220,7 +220,7 @@ func writeKeys(w io.Writer, n *store.NodeExtern) uint64 {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprintln(w, string(b))
```

Comment:
@hongchaodeng can you explain why do we need to change fprintf to fprintln in the commit message in detail?